### PR TITLE
README: update logo link

### PR DESCRIPTION
**Pull request by Mic92**
bors merge

### CODE REVIEW

**Comment by Mic92** [comment]
bors merge

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # nixos-anywhere - install nixos everywhere via ssh
 
-<img src="https://raw.githubusercontent.com/numtide/nixos-anywhere/5bdd3f3c442b923813e301b319290305353a3e7a/docs/logo.png" width="256" height="256">
+<img src="https://raw.githubusercontent.com/numtide/nixos-anywhere/main/docs/logo.png" width="256" height="256">
```
Comment (zimbatm) [inline README.md:3]: `docs/logo.png` is actually enough

Comment (Mic92) [reply README.md:3]: Not quite because one cannot resize images this way with Github markdown according to what I read.

Comment (zimbatm) [reply README.md:3]: See https://github.com/numtide/treefmt/blob/a3fe774af1e50c352f2cd9caf2fd634a88a69d0c/README.md?plain=1#L3

Comment (Mic92) [reply README.md:3]: Ah right. This is what you mean.

**Pull request merged**